### ISSUE TITLE
Unable to deploy injector only

### ISSUE THREAD

**Issue by herrbpl** [hashicorp/vault-helm#173]
Attempt to deploy following configuration:

```
global:
    enabled: false
  injector:
    # True if you want to enable vault agent injection.
    enabled: true
```
results following error from helm:
**Error: release vault-injector failed: no objects visited**

Expected result: 
- Injector Agent installed

Actual result:
- Deployment fails

Helm template version 0.3.2
Target Environment: Rancher v2.3.2

**Comment by ianks** [comment]
I was curious if it is possible to run the injector only since we already have a Vault server. Is this a supported use case?

**Comment by herrbpl** [comment]
Yes it can be run, I extracted injector part from Helm Chart and made it run standalone. Just needed to add additional parameter to specify Vault server. Hope this will be supported by official chart as well.

**Comment by jleechp-occm** [comment]
Would definitely be nice to have as a supported feature.  Multiple k8s clusters don't need multiple Vault instances as opposed to leveraging a centralized one.

**Comment by tvoran** [comment]
Hi there, this should be possible with the current chart as described in this thread: https://github.com/hashicorp/vault-k8s/issues/15

Adding an example of this kind of configuration to the docs is definitely on the todo list.

**Comment by lawliet89** [comment]
In case you are trying to find out how to opt out of the server portion of the chart only, you can find the information in this PR: https://github.com/hashicorp/vault-helm/pull/159

**Comment by tvoran** [comment]
Hi @herrbpl, sorry this was auto-closed without another response. We recently merged #207 which I think addresses your issue since it allows for deploying only the injector against an external vault. Please feel free to re-open if you have further questions.
